### PR TITLE
Master studio lpe

### PR DESCRIPTION
**Pull request by kebeclibre**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by kebeclibre** [comment]
https://pad.odoo.com/p/wowl_studio

File: addons/web/static/src/views/fields/field.js [hunk 1]
```diff
@@ -270,7 +267,7 @@ Field.parseFieldNode = function (node, models, modelName, viewType, jsClass) {
                 viewMode = "list";
             } else if (!views.list && views.kanban) {
                 viewMode = "kanban";
-            } else {
+            } else if (views.list && views.kanban) {
```
Comment (Polymorphe57) [inline addons/web/static/src/views/fields/field.js:270]: not sure about this

File: addons/web/static/tests/views/form/form_compiler_tests.js [hunk 2]
```diff
@@ -86,16 +86,16 @@ QUnit.module("Form Compiler", (hooks) => {
             </form>`;
         const expected = /*xml*/ `
             <OuterGroup>
-               <t t-set-slot="item_0" type="'item'" sequence="0" t-slot-scope="scope" itemSpan="1">
+               <t t-set-slot="item_0" type="'item'" sequence="0" t-slot-scope="scope" isVisible="true" itemSpan="1">
```
Comment (Polymorphe57) [inline addons/web/static/tests/views/form/form_compiler_tests.js:89]: don't like this change: we put a completely useless attribute it seems

**Comment by kebeclibre** [comment]
robodoo rebase-ff

**Comment by kebeclibre** [comment]
closed in favor of https://github.com/odoo/odoo/pull/101932

**Pull request closed without merging**